tests/helpers/index.js

```diff
@@ -0,0 +1,42 @@
+import {
+  setupApplicationTest,
+  setupRenderingTest,
+  setupTest,
+} from 'ember-qunit';
+
+// This file exists to provide wrappers around ember-qunit's / ember-mocha's
+// test setup functions. This way, you can easily extend the setup that is
+// needed per test type.
+
+// function setupApplicationTest(hooks, options) {
```

Comment:
We could fix and open it up as in one of the upgrade I  got it right, update the imports as follows 
`
import {
  setupApplicationTest as upstreamSetupRenderingTest,
  setupRenderingTest as upstreamSetupRenderingTest,
  setupTest as upstreamSetupTest,
} from 'ember-qunit';
`